src/site/filters/liquid.js

```diff
@@ -593,4 +589,47 @@ module.exports = function registerFilters() {
 
   liquid.filters.sortEntityMetatags = item =>
     item ? item.sort((a, b) => a.key.localeCompare(b.key)) : undefined;
+
+  liquid.filters.createEmbedYouTubeVideoURL = url => {
+    if (!url) {
+      return url;
+    }
+
+    if (!_.includes(url, 'youtu')) {
+      return url;
+    }
+
+    if (_.includes(url, 'embed')) {
+      return url;
+    }
+
+    return _.replace(url, 'youtu.be', 'youtube.com/embed');
```

Comment:
Nice and simple! I think I would have parsed the URL's domain with the node `url` module and way over-complicated it lol.